src/main/target/MATEKF722/target.h

```diff
@@ -75,6 +75,14 @@
 
 #define PITOT_I2C_BUS           BUS_I2C1
 
+#define USE_RANGEFINDER
```

Comment:
`USE_RANGEFINDER`, `USE_RANGEFINDER_HCSR04_I2C`, `USE_OPFLOW` and `USE_OPFLOW_MSP` are already defined for all targets having more than 256k flash, it is not needed.